src/models/Thought.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require("mongoose");
+const moment = require("moment");
+
+// Reaction schema
+
+const reactionSchema = new mongoose.Schema(
+  {
+    reactionId: {
+      type: mongoose.Schema.Types.ObjectId,
+      default: () => new mongoose.Types.ObjectId(),
+    },
+    reactionBody: {
+      type: String,
+      required: true,
+      trim: true,
+      minlength: 1,
+      maxlength: 280,
+    },
+    username: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+
+    createdAt: {
+      type: Date,
+      default: Date.now,
+      get: (createdAtVal) =>
+        moment(createdAtVal).format("MMM DD, YYYY [at] hh:mm a"),
```

Comment:
Great work! :) 
If the hw didn't specify to do this formatting in a getter function a good alternative would be to do this in a setter method- that way you only ever have to manipulate this date once (when you save it to the db) which could significantly cut down resource usage in the long run.